src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -354,10 +354,31 @@ TEST_F(ResumeCtrlTest, StartResumption_AppWithCommands) {
   ON_CALL(*mock_app_, help_prompt_manager())
       .WillByDefault(ReturnRef(*mock_help_prompt_manager_));
 
-  for (uint32_t cmd_id = 0, internal_id = 1; cmd_id < count_of_commands;
+  std::vector<application_manager::CommandsMap> command_vec;
+  for (uint32_t count = 0u; count <= count_of_commands; ++count) {
+    command_vec.emplace_back();
```

Comment:
@APipko Why do you need to call this function every  iteration? Can you use vector:reserve once instead? Use C++11 loops please